common/src/config.rs

```diff
@@ -12,6 +12,12 @@ pub struct EpochConfig {
     pub proto_server_addr: SocketAddr,
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct CassandraConfig {
+    // Can't be SocketAddr because we want to use a DNS name.
+    pub cql_addr: String,
```

Comment:
if there isn't a standard rust one already, can we define a HostPort struct and use it here?

Reply:
I'll take a stab at it!

Reply:
Great! could you move the formatting and serialization functions etc. to the end of the file to be with similar functions, so that the config types are not interrupted with this boilerplate?